pkg/deploy/configuration.go

```diff
@@ -150,8 +150,22 @@ func (cac *EnvironmentServiceConfigController) Configure(client Client, deployme
 	}
 	errs := []string{}
 	//configure for any environment services already deployed
+	// ensure not to call configure multiple times for instance when mongo replica set present
+	configured := []string{}
+	var isConfigured = func(name string) bool {
+		for _, val := range configured {
+			if val == name {
+				return true
+			}
+		}
+		return false
+	}
 	for _, s := range services {
 		serviceName := s.Labels["rhmap/name"]
+		if isConfigured(serviceName) {
```

Comment:
would it be better to do `configuredServices := map[string]bool`
```go
if _, ok := configuredServices[serviceName]; !ok {
    ...
}
```
And avoid the double iteration here?